extensions/nextdns/src/components/error-view.tsx

```diff
@@ -0,0 +1,10 @@
+import { Icon, List } from "@raycast/api";
+import { EmptyView } from "./empty-view";
+
+export function ErrorView() {
+  return (
+    <List>
+      <EmptyView title="Failed to load domains" icon={Icon.Warning}></EmptyView>
```

Comment:
non-blocking: can use the color red with the icon for failure.

Reply:
Updated with `tintColor` and added Action for opening preferences